containers/server-image/uyuni-configfiles-sync.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Uyuni configuration files sync to persistent volumes
```

Comment:
I think that instead of adding a new service we could enhance the existing one to also call this script at `uyuni-update-config.service´

It calls the script `/usr/sbin/uyuni-update-config`. It's a one time execution also. 

I would prefer to centralize all these calls in one service, that we can extend or refactor in the future.